docs/architecture/providers/parameterized.md

```diff
@@ -70,5 +119,15 @@ the `terraform` provider plugin (with a parameter of `aws:<version>` or similar,
 for example).
 :::
 
-(replacement-extension-providers)=
+(replacement-extension-parameterization)=
 ## Replacement and extension parameterization
+
+Currently, the only kind of parameterization is *replacement parameterization*.
+In this mode, once the provider has been parameterized, it will be used for that
+parameter only; and not without the parameter or with another parameter.
```

Comment:
```suggestion
Currently, the only kind of parameterization is *replacement parameterization*. 
In this mode, the parameterization of the provider _replaces_ the schema and any additional data the provider uses. Once the provider has been parameterized, it will be used for those parameters only; and not without the parameters or with other parameters.
```

The replacement of the schema, instead of managing additions, was a stumbling block for me initially.